### PR TITLE
A few more documentation around operation

### PR DESCRIPTION
**Pull request by franckcuny**

### CODE REVIEW

File: docs/operations/deployment.rst [hunk 1]
```diff
@@ -1,25 +1,92 @@
...
+a Bookkeeper cluster.
 
-Build
------
+Apache Zookeeper
+----------------
 
-To build DistributedLog, run:
+To run a DistributedLog ensemble, you'll need a set of Zookeeper
+nodes. There is no constraints on the number of Zookeeper nodes you
```
Comment (mgodave) [inline docs/operations/deployment.rst]: s/is/are/

File: docs/operations/deployment.rst [hunk 2]
```diff
@@ -1,25 +1,92 @@
...
+need. One node is enough to run your cluster, but for reliability
+purpose, you should run at least 3 nodes in production.
```
Comment (mgodave) [inline docs/operations/deployment.rst]: s/purpose/purposes/

File: docs/operations/deployment.rst [hunk 3]
```diff
@@ -1,25 +1,92 @@
...
+
+Since DistributedLog persist some meta-data in zookeeper, you'll need
```
Comment (mgodave) [inline docs/operations/deployment.rst]: s/persist/persists/

File: docs/operations/deployment.rst [hunk 4]
```diff
@@ -1,25 +1,92 @@
...
+to make sure your data are stored on persistent disks.
```
Comment (mgodave) [inline docs/operations/deployment.rst]: s/are/is/

File: docs/operations/deployment.rst [hunk 5]
```diff
@@ -1,25 +1,92 @@
...
+
+Apache Bookkeeper
+-----------------
+
+In a normal setup, you'll want at least a number of bookies equal to
+your replication factor plus one. If you choose to have 3 replicas,
+you'll want 4 bookies, that way if one of them goes down, you still
+have enough hosts for the quorum.
+
+The bookies are the storage layer in DistributedLog. They should run
```
Comment (mgodave) [inline docs/operations/deployment.rst]: Maybe move this sentence above the previous one.

File: docs/operations/deployment.rst [hunk 6]
```diff
@@ -1,25 +1,92 @@
...
+on hosts with persistent storage.
+
+Metadata
+~~~~~~~~
+
+Some metadata need to be created in Zookeeper before starting the
```
Comment (mgodave) [inline docs/operations/deployment.rst]: s/need/needs/

File: docs/operations/performance.rst [hunk 7]
```diff
@@ -1,4 +1,26 @@
 Performance Tuning
 ==================
 
-(describe how to tune performance, critical settings)
+We've found that this values are giving us the best performances (please refer to :doc`/configuration/core` for the meaning of each setting):
```
Comment (mgodave) [inline docs/operations/performance.rst]: We've found that the following settings have given us the best performance

**Comment by mgodave** [comment]
minor gramatical fixes, otherwise #shipit

**Comment by sijie** [comment]
@franckcuny this change might need rebase to master

**Pull request closed without merging**